xbmc/cores/VideoPlayer/Edl.cpp

```diff
@@ -124,12 +125,17 @@ bool CEdl::ReadEditDecisionLists(const std::string& strMovie, const float fFrame
   /*
    * PVR Recordings
    */
-  else if (URIUtils::IsPVRRecording(strMovie))
+  else if (fileItem.IsPVRRecording())
   {
     CLog::Log(LOGDEBUG, "%s - Checking for edit decision list (EDL) for PVR recording: %s",
       __FUNCTION__, strMovie.c_str());
 
-    bFound = ReadPvr(strMovie);
+    bFound = ReadPvr(fileItem);
+  } else if (fileItem.IsEPG()) {
```

Comment:
Coding guidelines: Please move the else clause to the next line and the opening curly brace to the line after the else clause.

<pre>
} 
else if (fileItem.IsEPG())
{
</pre>